src/qt/radiobut.cpp

```diff
@@ -39,11 +42,52 @@ bool wxRadioButton::Create( wxWindow *parent,
              const wxString& name)
 {
     m_qtRadioButton = new QRadioButton( parent->GetHandle() );
+
+    if ( (style & wxRB_GROUP) || (style & wxRB_SINGLE) )
```

Comment:
Not sure why do we need this test, it's a bit confusing to test for either `wxRB_GROUP` or `wxRB_SINGLE` first and then test again for the latter just below.

I'd rather structure this code like
```
if ( style & wxRB_SINGLE )
{
    m_lastGroup.erase(parent);
}
else if ( style & wxRB_GROUP )
{
    ... start a new group ...
}
else
{
    ... continue the previous group if any ...
}
```